integration/client/client.go

```diff
@@ -30,5 +31,9 @@ func NewImage(t *testing.T, namespace string) string {
 	if err != nil {
 		t.Fatal(err)
 	}
+	_, err = helper.BuilderClient(t, namespace).ImageCreate(context.Background(), image.ID, "")
```

Comment:
when we sync with darren's changes, i dont know if this will stay her permanently or not.